packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -89,6 +89,20 @@ async function handleGrant(config: ReleaseGoldConfig, currGrant: number) {
     '0x000000000000000000000000000000000000ce10',
   ]
 
+  const bytecode = await web3.eth.getCode(config.beneficiary)
+  if (bytecode !== '0x') {
+    const response = await prompts({
+      type: 'confirm',
+      name: 'confirmation',
+      message: `Beneficiary ${config.beneficiary} is a smart contract which might cause loss of funds if not properly configured. Are you sure you want to continue? (y/n)`,
+    })
+
+    if (!response.confirmation) {
+      console.info(chalk.yellow('Skipping grant due to user response'))
```

Comment:
Can we log the identifier as part of this? And/or write it (them) to a file?

Reply:
It's part of the previous message (which I just copied from the other prompt below)?